src/components/MultiSelect/icons/plus.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PlusIcon = props => {
+    const { className, style } = props;
+
+    return (
+        <svg className={className} style={style} width="14" height="14" viewBox="0 0 14 14">
+            <path
+                fill="#01B6F5"
```

Comment:
In order to do this icon customized with the theme let's use `currentColor` instead of a fixed color

Reply:
yes, we should use:
```fill="currentColor"```
in order to get the color from a parent element